tests/FlexTest.php

```diff
@@ -78,6 +97,27 @@ public function getRecordTests()
         ];
     }
 
+    /**
+     * dataset0: custom COMPOSER env is set, generates correct symfony lock
+     * dataset1: nothing overriden, just verify default names
+     * dataset2: composer = default, custom symfony.lock path
+     * @testWith ["composer.beta.json","", "composer.beta.lock", "symfony.beta.lock"]
+     *           ["","", "./composer.lock", "./symfony.lock"]
+     *           ["","symfony.beta.lock", "./composer.lock", "symfony.beta.lock"]
+     */
+    public function testSymfonyLock($composerEnv, $symfonyEnv, $expectedComposer, $expectedSymfony)
+    {
+        putenv("COMPOSER=" . $composerEnv);
+        putenv("SYMFONY_LOCKFILE_PATH=" . $symfonyEnv);
+
+        $flex = new Flex();
+        $r = $this->invokeMethod($flex, 'getLockFilePath', []);
+        $this->assertEquals($expectedComposer, $r->composer);
+        $this->assertEquals($expectedSymfony, $r->symfony);
+
+        putenv("COMPOSER=''");
+        putenv("SYMFONY_LOCKFILE_PATH=''");
+    }
```

Comment:
Missing newline after this one (I'm sorry for such reviews :rofl: )

Reply:
hahaha no worries, fixed! i am sorry for such laziness :D